qoe-metrics-dashboard/src/app/pages/overview/Overview.tsx

```diff
@@ -1,139 +1,186 @@
-import { useEffect, useState } from 'react';
+import { useContext, useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 
 import { ChevronLeft, ChevronRight } from '@mui/icons-material';
 import {
-  Box,
-  Button,
-  Checkbox,
-  Divider,
-  IconButton,
-  Typography,
+    Box,
+    Button,
+    Checkbox,
+    CircularProgress,
+    Divider,
+    IconButton,
+    Typography,
 } from '@mui/material';
 
-import ApiController from '../../api/ApiController';
+import { useReportList } from '../../api/ApiController';
+import { EnvContext } from '../../env.context';
+import { ESortingOrder } from '../../models/enums/shared/sorting-order.enum';
+import { TMetricsOverviewReport } from '../../models/types/responses/metrics-overview-report.interface';
 
 import './Overview.scss';
 
 const ROWS_PER_PAGE = 20;
 
 function Overview() {
-  const navigate = useNavigate();
-
-  const [metricsReports, setMetricsReports] = useState<any[]>([]);
-  const [currentPage, setCurrentPage] = useState(0);
-  const [selectedMetricsReports, setSelectedMetricsReports] = useState<
-    number[]
-  >([]);
-
-  useEffect(() => {
-    async function getMetricsReports(page: number): Promise<void> {
-      const reports = await ApiController.getMetricsReportsList(
-        page,
-        ROWS_PER_PAGE
-      );
-      if (reports.length === 0) {
-        setCurrentPage(currentPage - 1);
-      } else {
-        setMetricsReports(reports);
-      }
+    const navigate = useNavigate();
+
+    const envCtx = useContext(EnvContext);
+
+    // params for metrics report overview
+    const [provisionSessionIds, setProvisionSessionId] =
+        useState<RegExp>(/1-6/);
```

Comment:
die provisionSessionIds sind nur in der mock-AF 1-6 sonst beliebige Strings

Reply:
Ja das schon aber wir bauen das System halt für den MockServer und können es nicht testen für den normalen.
Außerdem ist das nur der initiale State den werde ich so anpassen wie er "für" uns am besten ist.